compatibility/BUILD

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "@daml//bazel_tools/client_server:client_server_test.bzl",
+    "client_server_test",
+)
+load("@os_info//:os_info.bzl", "is_linux")
+load("@bazel_tools//tools/python:toolchain.bzl", "py_runtime_pair")
+
+py_runtime(
+    name = "nix_python3_runtime",
+    interpreter = "@python3_nix//:bin/python",
+    python_version = "PY3",
+)
+
+py_runtime_pair(
+    name = "nix_python_runtime_pair",
+    py3_runtime = ":nix_python3_runtime",
+)
+
+toolchain(
+    name = "nix_python_toolchain",
+    exec_compatible_with = [
+        "@rules_haskell//haskell/platforms:nixpkgs",
+    ],
+    toolchain = ":nix_python_runtime_pair",
+    toolchain_type = "@bazel_tools//tools/python:toolchain_type",
+)
```

Comment:
`rules_nixpkgs` has [`nixpkgs_python_configure`](https://github.com/tweag/rules_nixpkgs/blob/d3c7bc94fed4001d5375632a936d743dc085c9a1/nixpkgs/nixpkgs.bzl#L418) which takes care of all that. Example use case: https://github.com/tweag/rules_haskell/blob/c625c53c3fe6644abb366bd868051d9e82f2a204/WORKSPACE#L209

Reply:
Oh fantastic! I’ve switched to that. I guess we can also switch to that for the main workspace? I had just copied what we did there. If so, I’ll do that in a separate PR.

Reply:
Yes, I thought I had already switched it over in `@daml`. Must have forgotten...